Content.Shared/_RMC14/Xenonids/Egg/XenoEggSystem.cs

```diff
@@ -229,13 +230,49 @@ private void OnXenoEggAfterInteract(Entity<XenoEggComponent> egg, ref AfterInter
     private void OnXenoEggActivateInWorld(Entity<XenoEggComponent> egg, ref ActivateInWorldEvent args)
     {
         // TODO RMC14 multiple hive support
-        if (!HasComp<XenoComponent>(args.User) || !HasComp<HandsComponent>(args.User))
+        if (!HasComp<XenoParasiteComponent>(args.User) && (!HasComp<XenoComponent>(args.User) || !HasComp<HandsComponent>(args.User)))
             return;
 
         if (Open(egg, args.User, out _))
             args.Handled = true;
     }
 
+    private void OnXenoEggInteractUsing(Entity<XenoEggComponent> egg, ref InteractUsingEvent args)
```

Comment:
This will also put them back in and delete them even if they are a player-controlled parasite, 13 doesn't even let you pick those up but for us it might be better to check if there is a player controlling the parasite, and if so to not run this code